oceanspy/compute.py

```diff
@@ -2833,6 +2834,43 @@ def salt_budget(od):
     return _ospy.OceanDataset(ds).dataset
 
 
+def find_cs_sn(thetaA, phiA, thetaB, phiB):
+    """
+    theta is the angle
```

Comment:
You should improve the description of this function. What does it do? Input and outputs. types.